.changeset/wild-fishes-carry.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react': patch
```

Comment:
I would say it is a minor change as we are adding a functionality that is backwards compatible. https://semver.org/#summary

Reply:
Perfect, I'll make the change here. I had put minor, but ended up being unsure and switched to patch.